schemas/EGA.ArrayAssay.json

```diff
@@ -0,0 +1,89 @@
+        "allOf": [
+          {
+            "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/object_coreIDs"
+          },
+          {
+            "properties": {
+              "accession": {
+                "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/EGA-ArrayAssay-id-pattern"
+              }
+            }
+          }
+        ]        
+      },
+
+      "arrayAssay_center": {
+        "type": "string",
+        "title": "Centername that performed the ArrayAssay",
+        "description": "The name of the center (e.g. 'EBI-TEST') responsible for the profiling by microarray (executed the ArrayAssay), if applicable (in case it’s different from the center submitting metadata)."
+      },
+
+      "arrayAssay_date": {
+        "type": "string",
+        "title": "Date of the ArrayAssay",
+        "description": "ISO date (format YYYY-MM-DD - e.g. '2021-05-15') when the ArrayAssay took place. If the protocols are long enough, the date shall be the day the ArrayAssay concluded.",
+        "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/EGA-ISO-date-YYYY-MM-DD-pattern"
+      },
+
+      "sample_number": {
+        "type": "integer",
+        "title": "Number of samples of the ArrayAssay",
+        "description": "Number of samples included in the Assay (i.e. pooled into one single microarray, labelled differently). One sample corresponds to one biological replicate [EFO:0002091] (it could be the genetic content from a single cell, a tissue… from a single individual or from several individuals). Shall not be mistaken for technical replicates [CHEBI:24432] being used several times (see https://www.ebi.ac.uk/seqdb/confluence/pages/viewpage.action?spaceKey=EGA&title=Sample+Representation)."
+      },
+
+      "sample_Labels": {
+        "type": "array",
+        "title": "Array of Sample-label pairs of the ArrayAssay",
+        "description": "Sample-Label pairs (format being 'EGAN[0-9]{11}:Label' - e.g. 'EGAN00000000001:Cy3') to know which samples used in this assay are labelled by which chemicals. Could be omitted if the array is of one single label and colour. #! Using an empty item list but defining the sample-label pairs as possible additionalItems we create the correct constraint: anything but a sample-label pair is rejected, but EGA can add as many as required.",
+        "additionalProperties": false,
+        "items": [],
+        "additionalItems": {
+          "type": "string",
+          "title": "Sample-label pair",
+          "pattern": "^EGAN\\d{11}:[a-zA-Z0-9 ]+$"
```

Comment:
As in the case of the ontologies, please, consider using 2 fields instead of concatenating the sample ID and the label with a colon. I believe this is clearer and easier to extend (to accept an `alias` and a `center_name`):
```json
{
   "sample_Labels": [ {
       "id": "EGAN00000000001", 
       "label": "Cy3"
   }, {
       "id": "EGAN00000000002",
       "label": "Biotin"
   }, {
       "alias": "my_sample_01",
       "center_name": "EBI-TEST",
       "label": "XXXXXX"
   }]
}
```